content/en/docs/installation/system-requirements.md

```diff
@@ -0,0 +1,90 @@
+---
+title: System Requirements for Armory Enterprise
+linkTitle: System Requirements
+description: >
+  Armory Enterprise is a collection of services that run in a Kubernetes cluster. In addition to the cluster, other requirements, such as storage, need to be met to run Armory Enterprise for production environments.
+---
+
+The requirements described on this page are meant as a minimum starting point for installing Armory Enterprise. You may need to increase the resources based on the number of pipelines and executions.
+
+## Installation targets
+
+> This section defines where you can run Armory Enterprise, not where you can deploy your applications. For information about where you can deploy applications to, see the [Product Compatibility Matrix]({{< ref "armory-enterprise-matrix#deployment-targets" >}}).
+
+Armory Enterprise can be installed on any [certified Kubernetes cluster](https://www.cncf.io/certification/software-conformance/).
+
+* **Minimum version**: Some TBD version
+* **Maximum version**: Some TBD version
```

Comment:
1.20 as of this writing (upper bound for EKS)

I think long-term we should follow what EKS supports until we see customers overwhelmingly on another solution